drivers/dsp0401/dsp0401.c

```diff
@@ -238,12 +238,12 @@ void dsp0401_scroll_text(const dsp0401_t *dev, char *text, uint16_t delay)
     for (unsigned i = 0; i < strlen(text); ++i) {
         _shift_char(dev, text[i]);
         _latch(dev);
-        xtimer_usleep((uint32_t)(delay * US_PER_MS));
+        xtimer_msleep((uint32_t)(delay));
```

Comment:
```suggestion
        xtimer_msleep(delay);
```

`uint16_t` can be implicitly and without loss promoted to `uint32_t`

Reply:
my sed script was not so clever :wink: